src/cmocka/rnp_tests_exportkey.c

```diff
@@ -50,7 +50,7 @@ rnpkeys_exportkey_verifyUserId(void **state)
     rnp_assert_ok(rstate, rnp_key_store_load_keys(&rnp, 1));
```

Comment:
Sorry this comment is about utils.h but I can't comment there.
Why do we move utils.h to "include" ?

My understanding was that this directory contains only files that are public API. Having public API in separated definitely is good idea. Now utils.h contains stuff which should be kept internal to rnp. We could have separated directory called "common" or "utils" which would contains stuff that's commonly used by different components but are not public API.
We should be careful about what we make public API.